opencog/nlp/learn/misc-scripts/opencog-en.conf

```diff
@@ -32,11 +32,11 @@ ANSI_SCM_PROMPT       = "[0;34mguile-en[1;34m> [0m"
 # codes
 ANSI_ENABLED	       = true
 
-MODULES               = opencog/modules/libbuiltinreqs.so,
-                        opencog/modules/libPersistModule.so,
-                        opencog/modules/libQueryModule.so,
-                        opencog/modules/libscheme-shell.so,
-                        opencog/modules/libnlp-types.so
+MODULES               = opencog/cogserver/modules/libbuiltinreqs.so,
+                        opencog/cogserver/modules/libPersistModule.so,
+                        opencog/cogserver/modules/libQueryModule.so,
+                        opencog/cogserver/modules/libscheme-shell.so,
+                        opencog/cogserver/modules/libnlp-types.so
 
```

Comment:
Please don't!  Seriously, this completely breaks the server!  You need to test before making these changes.